templates/Includes/DocumentationSidebar.ss

```diff
@@ -3,9 +3,9 @@
 		$DocumentationSearchForm
 		
 		<ul class="nav">
+			<li><a href="$Link" class="top">Home</a></li>
 
 			<% loop Menu %>
-				<li><a href="$Link" class="top">Home</a></li>
```

Comment:
I'll double check again in about an hour but in my testing it would put say "Home" followed right below by say "doc viewer" then home again and the next module.